demisto_sdk/commands/pre_commit/tests/pre_commit_test.py

```diff
@@ -19,6 +20,25 @@
     Path(git_path()) / "demisto_sdk" / "commands" / "pre_commit" / "tests" / "test_data"
 )
 
+PYTHON_VERSION_TO_FILES = {
+    "3.8": {PosixPath("Packs/Pack1/Integrations/integration1/integration1.py")},
+    "3.9": {PosixPath("Packs/Pack1/Integrations/integration2/integration2.py")},
+    "3.10": {PosixPath("Packs/Pack1/Integrations/integration3/integration3.py")},
+}
+
+
+def create_hook(hook: dict):
+    """
+    This function mocks hook as he returns in _get_hooks() function
+    """
+    repo_and_hook: dict = {"repo": {"repo": "repo", "hooks": [hook]}}
+    repo_and_hook["hook"] = repo_and_hook["repo"]["hooks"][0]
+    return repo_and_hook
+
+
+def files_for_hook(files):
```

Comment:
see my other comment (taking the equivalent of this out from the Hook class), we can import it instead of rewriting it.